R/bs-dependencies.R

```diff
@@ -182,12 +181,15 @@ bs_theme_dependencies <- function(
 #' @param .sass_args A list of additional arguments to pass to
 #'   [sass::sass_partial()].
 #' @inheritParams htmltools::htmlDependency
-#' @references
-#' <https://rstudio.github.io/bslib/articles/custom-components.html>
 #'
+#' @references
+#'   * [Theming: Custom components](https://rstudio.github.io/bslib/articles/custom-components/index.html)
+#'     gives a tutorial on creating a dynamically themable custom component.
 #'
 #' @return `bs_dependency()` returns an [htmltools::htmlDependency()] and
 #'   `bs_dependency_defer()` returns an [htmltools::tagFunction()]
+#'
+#' @family Bootstrap theme functions
```

Comment:
Feels like we could split this into multiple families?

* Custom theming: `bs_theme()`, `bs_add_variables()`, `bs_theme_preview()`, etc.
* Programming with themes: `bs_global_theme()`, `bs_current_theme()`, `bs_remove()`, etc.
* Dynamically themable components: `bs_dependency_defer()`, `bs_dependency()`, etc.

Reply:
We could. I didn't do this initially because the way the functions are documented, each family will list only one or two other functions. The `@family` tag results in `Other {family}: _____` in the docs, and only references the primary function in the documentation.

Inside the docs of any of the above functions we'd link to their siblings as needed, much like we have in our pkgdown docs, but I saw `@family` as a way to jump out of the local group and into other functions that are related to theming.

Reply:
Yea, I can see where you're coming from, sounds good